modules/imgproc/src/drawing.cpp

```diff
@@ -1879,6 +1880,45 @@ void rectangle( InputOutputArray img, Rect rec,
         rectangle( img, rec.tl(), rec.br() - Point(1<<shift,1<<shift),
                    color, thickness, lineType, shift );
 }
+void dash_rectangle(InputOutputArray _img, Point pt1, Point pt2,
+    const Scalar& color, int thickness = 1,
+    int lineType = 8, int shift = 0,int gap=5) {
```

Comment:
Please use enum values https://docs.opencv.org/master/d6/d6e/group__imgproc__draw.html#gaf076ef45de481ac96e0ab3dc2c29a777 instead of int `8`